files/run_worker.sh

```diff
@@ -12,40 +14,34 @@ source /usr/bin/setup_env_in_openshift.sh
 
 mkdir -p "${PACKIT_HOME}/.ssh"
 chmod 0700 "${PACKIT_HOME}/.ssh"
-pushd "${PACKIT_HOME}/.ssh" || exit
+pushd "${PACKIT_HOME}/.ssh"
 install -m 0400 /packit-ssh/id_rsa .
 install -m 0400 /packit-ssh/id_rsa.pub .
 if [[ -f /packit-ssh/config ]]; then install -m 0400 /packit-ssh/config .; fi
 grep -q pkgs.fedoraproject.org known_hosts || ssh-keyscan pkgs.fedoraproject.org >>known_hosts
-popd || exit
+popd
 
-DEFAULT_CELERY_COMMAND="worker"
 # Whether to run Celery worker or beat (task scheduler)
-CELERY_COMMAND="${CELERY_COMMAND:-$DEFAULT_CELERY_COMMAND}"
-
-if [[ "${CELERY_COMMAND}" == "beat" ]]; then
+if [[ "${CELERY_COMMAND:=worker}" == "beat" ]]; then
```

Comment:
had to read definition of `${foo:=bar}` to understand this:

```
${parameter:=word}
If parameter is unset or null, the expansion of word is assigned to parameter. The value of parameter is then substituted. Positional parameters and special parameters may not be assigned to in this way.
```

https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

+1